### PR TITLE
 Patch - SUPEE-10415 / 1.9.3.7

### PR DESCRIPTION
**Pull request by spinsch**
**Release Notes**
http://devdocs.magento.com/guides/m1x/ce19-ee114/ce1.9_release-notes.html#ce19-1936

**Security Patch-Notes**
https://magento.com/security/patches/supee-10415

### CODE REVIEW

File: app/code/core/Mage/Customer/Model/Customer.php [hunk 1]
```diff
@@ -73,6 +73,11 @@ class Mage_Customer_Model_Customer extends Mage_Core_Model_Abstract
      */
     const MINIMUM_PASSWORD_LENGTH = 6;
 
+    /**
+     * Maximum Password Length
+     */
+    const MAXIMUM_PASSWORD_LENGTH = 256;
```
Comment (xf-) [inline app/code/core/Mage/Customer/Model/Customer.php]: Never used

Comment (drobinson) [reply app/code/core/Mage/Customer/Model/Customer.php]: It should be - it's used here: https://github.com/OpenMage/magento-mirror/blob/a5ad2ee47599400ef0066562315b300c7f581938/app/code/core/Mage/Customer/Model/Customer.php

@spinsch Where there any issues applying the patch? Or is this an issue with the patch released by Magento?

Comment (spinsch) [reply app/code/core/Mage/Customer/Model/Customer.php]: Sorry. It was my fault - the release from Magento is ok.

**Comment by drobinson** [comment]
@spinsch Seems it's still missing some things - see diff between these diffs

https://patch-diff.githubusercontent.com/raw/OpenMage/magento-lts/pull/393.diff
vs
https://github.com/OpenMage/magento-mirror/commit/a5ad2ee47599400ef0066562315b300c7f581938.diff

Typically what I do to apply these patches is grab the latest diff from Magento-Mirror and apply it directly to this repo - this usually results in fewer conflicts during the patch process.

**Comment by spinsch** [comment]
@drobinson: sorry for my mistakes. I‘ve downloaded the official archives and compared them manually to make the patch. It was a bad idea 😌

The current version is made now how you have explained it, namely with a diff of the magento-mirror repository and the merging in our repository. I learn each day and will do it better for the future.

**Comment by drobinson** [comment]
Looks good - much appreciated @spinsch 

> I learn each day and will do it better for the future.

I think we all hope to do the same!

**Pull request merged**